README.md

```diff
@@ -281,6 +281,7 @@ The following model architectures, tasks and device distributions have been vali
 | Baichuan2 | <li>DeepSpeed</li> | <li>Single card</li> | <li>[language modeling](https://github.com/huggingface/optimum-habana/tree/main/examples/language-modeling)</li><li>[text generation](https://github.com/huggingface/optimum-habana/tree/main/examples/text-generation)</li> |
 | DeepSeek-V2 | :heavy_check_mark: | :heavy_check_mark: | <li>[text generation](https://github.com/huggingface/optimum-habana/tree/main/examples/text-generation)</li> |
 | DeepSeek-V3 |   | :heavy_check_mark: | <li>[text generation](https://github.com/huggingface/optimum-habana/tree/main/examples/text-generation)</li> |
+| Moonlight |   | :heavy_check_mark: | <li>[text generation](https://github.com/huggingface/optimum-habana/tree/main/examples/text-generation)</li> |
```

Comment:
Since the architecture is exactly the same as Deepseek-V3, let's rather change the line right above for Deepseek-V3 into `Deepseek-V3 / Moonlight`, similarly to Llama and Granite for instance: https://github.com/huggingface/optimum-habana/blob/55c456ab12f271a4ff373c5c6cdfc2cc480d0535/README.md?plain=1#L244

Reply:
done